packages/generator-electrode/README.md

```diff
@@ -42,6 +42,47 @@ Some common ones:
   - `gulp server-prod` - start server in production mode
   - `gulp check` - run unit tests with coverage
 
+## Generating a React Component
+  Install the generator if you haven't already:
+
+  ```bash
+  npm install -g generator-electrode
```

Comment:
That's the easiest way, since you don't have your own project yet when you need to use the generator.